src/manifolds/GeneralizedGrassmann.jl

```diff
@@ -127,7 +127,7 @@ end
     distance(M::GeneralizedGrassmann, p, q)
 
 Compute the Riemannian distance on [`GeneralizedGrassmann`](@ref)
-manifold `M`$= \mathrm{Gr}(n,k,B)$.
+manifold `M```= \mathrm{Gr}(n,k,B)``.
```

Comment:
Does this work without some spaces?

Reply:
oh, I am not sure, if it would, that would of course be great, but maybe it does not.